notebooks/8_combine_dataframes.ipynb

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
<p>I suggest splitting this into 3 separate print statements (on 3 lines in the same cell). Currently, it's a bit hard for me to visually link each object name to its shape.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>